src/useRefs/useRefs.stories.tsx

```diff
@@ -19,8 +20,8 @@ type DemoComponentRefs = Refs<{
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 function DemoComponent(): ReactElement {
-  const listItemsRef = useRef([]);
-  const keyListRef = useRef([]);
+  const listItemsRef = useRef(createRefArray());
+  const keyListRef = useRef(createRefArray());
```

Comment:
This is not necessary anymore, a new array is created using `createRefArray` when the value is `null` in the `arrayRef` util.

Reply:
But what if you pass an array of your own? That's still possible, right? Then you'll lose the trimming functionality without realizing.

Reply:
Yes, but I don't think we should overwrite the initial values because that feels like an unexpected side effect. If someone does provide a plain array there's probably a good reason to do so. It's still possible to create an array with the util if a self-trimming array is needed.

Reply:
Moved the trimming to the `arrayRef` util